common/lib/plugins/aurora_initial_connection_strategy_plugin.ts

```diff
@@ -191,16 +191,13 @@ export class AuroraInitialConnectionStrategyPlugin extends AbstractConnectionPlu
       readerCandidate = null;
 
       try {
+        readerCandidateClient = await connectFunc();
```

Comment:
In some case this client can be disposed without proper closing it. For example, at line 224, a new client is created and that overrides existing client.